index.d.ts

```diff
@@ -0,0 +1,10 @@
+interface ClearablePromise<T> extends Promise<T> {
+  clear (): void
+}
+
+declare const delay: {
+  <T = never>(ms: number, value?: T): ClearablePromise<T>
+  reject(ms: number, reason?: any): ClearablePromise<never>
+}
+
+export = delay
```

Comment:
Semicolon

Reply:
Can you use `export default delay;`?

Reply:
`export default` isn't going to work because it's just CJS. So if you would write `module.exports.default = delay`, we could use `export default delay;` here as well.